shared/types/field/dateField.ts

```diff
@@ -26,5 +27,5 @@ export type DateValidationOptions = {
 export interface DateFieldBase extends MyInfoableFieldBase {
   fieldType: BasicField.Date
   dateValidation: DateValidationOptions
-  invalidDaysOfTheWeek?: DaysOfTheWeek[] | null
+  invalidDays?: InvalidDaysOptions[]
```

Comment:
comment: no more null? actually looks correct. can i have a refresh on why null was there before?

Reply:
I added null because I thought previously if no checkboxes were selected, it would be a null value instead of empty array. But since we added a checkbox group to group all the checkboxes, the default value would be an empty array instead of null if no checkbox was selected.